docs/proposals/20191030-machine-health-checking.md

```diff
@@ -158,20 +164,26 @@ type target struct {
 
 - Calculate the number of unhealthy targets.
 - Compare current number against `maxUnhealthy` threshold and temporary short circuits remediation if the threshold is met.
-- Trigger remediation for unhealthy targets i.e request machines for deletion.
+- Annotates unhealthy target machines as described above.
 
 Out of band:
-- The owning controller e.g machineSet controller reconciles to meet number of replicas and start the process to bring up a new machine/node.
+- The owning controller observes the annotation and is responsible to remediate the machine.
+- The owning controller performs any pre-deletion tasks required.
+- The owning controller MUST delete the machine.
```

Comment:
Why must the machine be deleted?

Reply:
For the purpose of this effort, which is mainly to separate the responsibilities between controllers, we want to make sure the expected outcomes are spelled out.

In the future, when new proposals come out and have new strategies, we'll redesign this.